tests/test_samples.py

```diff
@@ -1223,13 +1235,15 @@ def test_samples_dot_plot():
 
     axes = samples.plot.kde_2d('x0', 'x1')
     assert len(axes.collections) == 5
-    assert axes.get_xlabel() == 'x0'
-    assert axes.get_ylabel() == 'x1'
+    assert axes.get_xlabel() == '$x_0$'
+    assert axes.get_ylabel() == '$x_1$'
```

Comment:
Following up on my above comment, we should add a test for:
```python
axes = samples.drop_labels().plot.kde_2d('x0', 'x1')
assert axes.get_xlabel() == 'x0'
assert axes.get_ylabel() == 'x1'
```

Reply:
fixed by [4ea5195](https://github.com/handley-lab/anesthetic/pull/304/commits/4ea5195e006f6f837636f3b3d6815808f5df9110), will add a test